src/FMBot.Persistence.EntityFrameWork/FMBot.Persistence.EntityFrameWork.csproj

```diff
@@ -7,9 +7,13 @@
     <StartupObject />
     <LangVersion>9.0</LangVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Properties\**" />
+    <EmbeddedResource Remove="Properties\**" />
+    <None Remove="Properties\**" />
+  </ItemGroup>
```

Comment:
If you're removing something from the project does it even have to be in the `.csproj` at all?

Reply:
Yes, No, Both!

Its a _feature_ visual studio forces for people who still have these folders locally. Having it in here vs having it not in here has about the same effect.

Your choice what you want to do with it